hasjob/views/campaign.py

```diff
@@ -180,7 +180,7 @@ def action_new(self, **kwargs):
     @route('views.csv')
     def view_counts(self, **kwargs):
         campaign = self.obj
-        timezone = current_auth.actor.timezone if current_auth else 'UTC'
+        timezone = current_auth.actor.timezone if current_auth.not_anonymous else 'UTC'
```

Comment:
This one can be `current_auth.actor.timezone if current_auth else 'UTC'` provided the AnonUser object reports a timezone (which it should).

Reply:
We're not setting and using `g.anon_user` until we close that other PR that moved AnonUser to mouse interaction. So gonna leave it like this for now and fix it when I close that other PR.